token-metadata/js/README.md

```diff
@@ -0,0 +1,61 @@
+# `@solana/spl-token-metadata`
+
+A TypeScript interface describing the instructions required for a program to implement to be considered a "token-metadata" program for SPL token mints. The interface can be implemented by any program.
+
+## Links
+
+- [TypeScript Docs](https://solana-labs.github.io/solana-program-library/token/js/)
```

Comment:
Gotta update this path

Reply:
Changed to:
`https://solana-labs.github.io/solana-program-library/token-metadata/js/`